### PR TITLE
fix: Add missing fields from PyPi upstream change

### PR DESCRIPTION
**Pull request by BruRdgz**

### CODE REVIEW

File: src/letsbuilda/pypi/models/models_json.py [hunk 1]
```diff
@@ -114,6 +114,8 @@ class Info:
     version: str
     yanked: bool
     yanked_reason: str | None
+    license_expression: str | None
+    license_files: str | None
```
Comment (Robin5605) [inline src/letsbuilda/pypi/models/models_json.py]:
```suggestion
    license_files: list[str] | None
```

**Review by Robin5605 — CHANGES_REQUESTED** [review]
also needs to be added the dynamic field

File: src/letsbuilda/pypi/models/models_json.py (hunk 1 above)
Comment (shenanigansd) [inline src/letsbuilda/pypi/models/models_json.py]:
```suggestion
    license_files: list[str] | None
```
https://github.com/pypi/warehouse/pull/16949/files#diff-730d8e751957f97fab26f9b37db88c03b058ad3d009e6a15de85cf14f587f608

**Review by shenanigansd — CHANGES_REQUESTED** [review]

**Comment by BruRdgz** [comment]
Shen did the fixes upstream, so yay 👍

**Pull request closed without merging**